src/backend/cuda/math.hpp

```diff
@@ -11,30 +11,28 @@
 
 #ifdef __CUDACC_RTC__
 
-#define CUDART_INF_F __int_as_float(0x7f800000)
-#define CUDART_INF __longlong_as_double(0x7ff0000000000000ULL)
 #define STATIC_ inline
 
 #else  //__CUDACC_RTC__
 
 #include <common/defines.hpp>
-#include <common/half.hpp>
-#include <af/defines.h>
 
 #ifdef __CUDACC__
 #include <cuda_runtime_api.h>
-#include <math_constants.h>
 #endif  //__CUDACC__
 
 #include <algorithm>
 #include <limits>
 
 #endif  //__CUDACC_RTC__
 
-#include <cuda_fp16.h>
+#include <backend.hpp>
+#include <common/half.hpp>
```

Comment:
I don't think this is needed here.

Reply:
It;s not something I added newly here I think. I just moved them out of guards since they are handled in nvrtc buildKernel helper now.